src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    public void doWork() {
+        System.out.println("Truck is now working.");

```

Comment:
The message printed should be more informative and indicate that it is a 'Truck' that is working. For example: 'Truck started its work.'